src/test/scala/io/iohk/ethereum/network/discovery/PeerDiscoveryManagerSpec.scala

```diff
@@ -1,215 +0,0 @@
-package io.iohk.ethereum.network.discovery
-
-import java.net.{InetAddress, InetSocketAddress}
-import java.time.{Clock, Instant, ZoneId}
-import akka.actor.ActorSystem
-import akka.testkit.{TestActorRef, TestProbe}
-import akka.util.ByteString
-import com.miguno.akka.testing.VirtualTime
-import io.iohk.ethereum.NormalPatience
-import io.iohk.ethereum.blockchain.sync.EphemBlockchainTestSetup
-import io.iohk.ethereum.network.discovery.DiscoveryListener._
-import io.iohk.ethereum.network.discovery.PeerDiscoveryManager.{DiscoveryNodeInfo, PingInfo}
-import io.iohk.ethereum.nodebuilder.{NodeKeyBuilder, SecureRandomBuilder}
-import io.iohk.ethereum.rlp.RLPEncoder
-import io.iohk.ethereum.utils.{Config, NodeStatus, ServerStatus}
-import java.util.concurrent.atomic.AtomicReference
-import org.scalamock.scalatest.MockFactory
-import org.scalatest.concurrent.ScalaFutures
-import scala.util.Success
-import org.scalatest.flatspec.AnyFlatSpec
-import org.scalatest.matchers.should.Matchers
-
-class PeerDiscoveryManagerSpec
```

Comment:
Should we add tests for our new PeerDiscoveryManager?

Reply:
I don't know, the actor doesn't have too much logic in it, so I thought I'd leave it up to you if you're happy with the three of us having seen it, and tested it with mordor/etc, or you'd prefer having unit tests with a mock service.

Reply:
What would be cool if there was some end-to-end test that start and least a couple of Mantis nodes that discovery each other, so we know the service builder is correct as well as the actor.

Reply:
Added unit tests.

Reply:
That would be cool... I'm not sure if we had that on Midnight or not :thinking:

Reply:
Kinda, it builds the project (although I think that could be skipped) and starts the node as subprocesses to test the wallet. Not sure if multiple nodes are tested though, probably not.